GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -731,6 +731,20 @@ namespace gdjs {
       this._pausedSounds.length = 0;
     }
 
+    interpolateAudioTo(
+      channel: integer,
+      toVolume: float,
+      timeOfFade: integer
+    ): this {
+      const sound = this.getSoundOnChannel(channel);
+      if (sound) {
+        sound.fade(sound.getVolume(), toVolume, timeOfFade);
+      } else {
+        logger.error('cannot interpolate audio on non-existing channel');
+      }
+      return this;
+    }
+
```

Comment:
```suggestion
```

This code is redundant and inconsistent with the rest of the functions for sound manipulation.